README.md

```diff
@@ -1,2 +1,57 @@
-# opencost-plugins
-Plugins for extending OpenCost with new Custom Cost Sources
+# OpenCost Plugins
+
+OpenCost plugins make extending [OpenCost](https://github.com/opencost/opencost)’s coverage into new external cost sources (monitoring, data platforms, cloud services, and other SaaSes) available to the open source community. They allow for the ingestion of arbitrary cost data into OpenCost through the conformance of the [FOCUS spec](https://focus.finops.org/).
```

Comment:
Since FOCUS doesn't have SaaS coverage yet, could you explain a little bit what "conformance" means? Like are we using their terminology for the fields in the SaaS bill that happen to match (ie. "BillingAccountName")? Perhaps we should emphasize that we're attempting to adhere to incomplete standards and will continue to adapt? Sorry for being pedantic.

Reply:
Pedantic is exactly how we should be! I'll get to work on clarifying based on your notes.